internal/namespaces/billing/v2alpha1/custom_billing_download.go

```diff
@@ -0,0 +1,239 @@
+package billing
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"reflect"
+	"strings"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/scaleway/scaleway-cli/v2/internal/core"
+	"github.com/scaleway/scaleway-cli/v2/internal/interactive"
+	"github.com/scaleway/scaleway-cli/v2/internal/terminal"
+	billing "github.com/scaleway/scaleway-sdk-go/api/billing/v2alpha1"
+)
+
+const (
+	invoiceDefaultPrefix = "scaleway-invoice"
+)
+
+type billingDownloadRequest struct {
+	billing.DownloadInvoiceRequest
+	// extra arguments
+	FilePath     string
+	ForceReplace bool
+}
+
+func fileNameWithoutExtTrimSuffix(fileName string) string {
+	return strings.TrimSuffix(fileName, filepath.Ext(fileName))
+}
+
+func buildDownloadCommand(command *core.Command) *core.Command {
+	command.ArgsType = reflect.TypeOf(billingDownloadRequest{})
+	command.ArgSpecs = core.ArgSpecs{
+		{
+			Name:       "invoice-id",
+			Short:      `Invoice ID`,
+			Required:   true,
+			Deprecated: false,
+			Positional: false,
+		},
+		{
+			Name:       "file-path",
+			Short:      `Wanted file path`,
+			Required:   false,
+			Deprecated: false,
+			Positional: false,
+			Default:    core.DefaultValueSetter("/tmp/"),
```

Comment:
Can we use local directory by default? "tmp" does not exist on windows
```suggestion
			Default:    core.DefaultValueSetter("./"),
```

Reply:
If what you want is a temporary directory, have a look at `core.ExtractCacheDir`